src/test/java/bio/terra/janitor/db/JanitorDaoTest.java

```diff
@@ -87,6 +89,23 @@ public void serializeCloudResourceUid() {
     return jdbcTemplate.queryForMap(sql, params);
   }
 
+  /**
+   * Query by CloudResourceUid but not use the default serialize method which includes null value to
+   * verify query works.
+   */
+  public static Map<String, Object> queryTrackedResourceNonNull(
+      NamedParameterJdbcTemplate jdbcTemplate, CloudResourceUid resourceUid)
+      throws JsonProcessingException {
+    ObjectMapper mapper =
+        new ObjectMapper().setSerializationInclusion(JsonInclude.Include.NON_NULL);
+    String sql =
+        "SELECT id, resource_uid::text, resource_type, creation, expiration, state FROM tracked_resource WHERE resource_uid::jsonb @> :resource_uid::jsonb";
```

Comment:
'@>' is not equality, its "contains top level fields." If this doesn't pass for '=' operator, then I don't think this works and we should use Include.NON_NULL so that we can add more resource types without messing up equality.
https://www.postgresql.org/docs/9.5/functions-json.html